packages/patrol/lib/src/custom_finders/patrol_tester.dart

```diff
@@ -637,16 +638,22 @@ class PatrolTester {
   }
 }
 
-/// Specifies how methods such as [PatrolTester.tap] or [PatrolTester.enterText] handle pumping, i.e rendering new frames.
+/// Specifies how methods such as [PatrolTester.tap] or [PatrolTester.enterText] handle pumping, i.e. rendering new frames.
 ///
 /// It's usually useful when dealing with situations involving finite and infinite animations.
 enum SettlePolicy {
-  /// When pumping should be performed, [PatrolTester.pump] will be called.
+  /// [PatrolTester.pump] is used when pumping.
+  ///
+  /// This renders a single frame. If some animations are currently working, they will move forward by a single frame.
   noSettle,
 
-  /// When pumping should be performed, [PatrolTester.pumpAndSettle] will be called.
+  /// [PatrolTester.pumpAndSettle] is used when pumping.
+  ///
+  /// This keeps on rendering new frames until there are no frames pending or timeout is reached. Throws an exception if timeout has been reached.
```

Comment:
Let's specify the type of the exception that's thrown.

IIRC it's [WaitUntilVisibleTimeoutException](https://pub.dev/documentation/patrol/latest/patrol/WaitUntilVisibleTimeoutException-class.html) - but not sure about it. BTW this name might not be good anymore.

Reply:
ping

Reply:
It actually throws `FlutterError('pumpAndSettle timed out')` :(

Reply:
Ah, right. Let's document it anyway, this is useful info.

Reply:
Already done